client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+        "headerGotoLocations": "Eingestellte Zielpositionen",
+        "headerForbiddenMarkers": "Verbotsmarkierungen",
+        "noZonesConfigured": "Bisher wurden keine Zonen eingestellt.",
+        "noSpotsConfigured": "Bisher wurden keine Zielpunkte eingestellt.",
+        "editForbiddenZones": "Einstellen von Verbotszonen",
+        "noPersistentData": "Dauerhafte Daten sind abgeschaltet oder nicht verfügbar. Es können keine Verbotsmarkierungen bearbeitet werden.",
+        "addButton": "Hinzufügen",
+        "renameButton": "Umbenennen",
+        "deleteButton": "Löschen",
+        "newZoneName": "Zonenname spezifizieren ...",
+        "newSpotName": "Markierungsname spezifizieren...",
+        "enterZoneName": "Bitte einen Zonenamen eingeben",
+        "enterSpotName": "Bitte einen Markierungsnamen eingeben",
+        "renameZone": "Name für neue Zone \"{{zone}}\":",
+        "renameSpot": "Name für neue Markierung \"{{spot}}\":",
+        "confirmDeleteSpot": "Soll die Markierung \"{{spot}}\" wirklich gelöscht werden?",
+        "confirmDeleteZone": "Soll die Zone \"{{zone}}\" wirklich gelöscht werden?",
+        "zonesPageBack": "Zonen",
+        "zoneEditTitle": "Bearbeiten von Zone: {{zone}}",
+        "tooManySubzones": "Es können nicht mehr al 5 Unterzonen in einer Zone gesetzt werden.",
+        "tooManyMarkers": "Es können keine weitere Verbotsmarker gesetzt werden!",
+        "zonesSavedOK": "Zone erfolgreich gespeichert!",
+        "spotsPageBack": "Markierung",
+        "spotEditTitle": "Bearbeiten von Markierung: {{spot}}",
+        "spotsSavedOK": "Markierung erfolgreich gespeichert!",
+        "forbiddenMarkersPageBack": "Verbotsmarkierungen",
+        "forbiddenMarkersTitle": "Markierungen bearbeiten",
+        "forbiddenMarkersSavedOK": "Verbotsmarkierungen erfolgreich gespeichert!"
+    },
+    "control": {
+        "startButton": "Manuelle Kontrolle starten",
+        "stopButton": "Manuelle Kontrolle beenden"
+    },
+    "settings": {
+        "title": "Einstellungen",
+        "infoTitle": "Informationen",
+        "infoDesc": "Geräteinformationen anzeigen",
+        "timersTitle": "Zeitsteuerung",
+        "timersDesc": "Einstellen der Zeitabhängigen Steuerung",
+        "carpetModeTitle": "Teppichmodus",
+        "carpetModeDesc": "Teppichmodus einstellen",
+        "persistentDataTitle": "Dauerhafte Daten",
+        "persistentDataDesc": "Einstellen des Labormodus um virtuelle Wände, etc. zu aktivieren.",
+        "consumablesTitle": "Verbrauchsmaterialien",
+        "consumablesDesc": "Ansehen/Zurücksetzten von Verbrauchszählern",
+        "cleaningHistoryTitle": "Reinigungshistorie",
+        "cleaningHistoryDesc": "Ansehen der Reinigungshistorie",
+        "wifiTitle": "W-LAN",
+        "wifiDesc": "Ansehen/Einstellen der W-LAN Details und Einstellungen",
+        "soundAndVoiceTitle": "Ton & Stimme",
+        "soundAndVoiceDesc": "Ändern der Lautsärke und die Sprache des Roboters",
```

Comment:
"soundAndVoiceDesc": "Ändern der Lautstärke und der Sprache des Roboters",
(In the final file the "t" in "Lautstärke" is still missing)